app/assets/stylesheets/components/_secondary-navigation.scss

```diff
@@ -0,0 +1,94 @@
+.app-c-secondary-navigation {
+  @include govuk-font(19);
+  @include govuk-responsive-margin(6, "bottom");
+
+  display: block;
+
+  @include govuk-media-query($from: tablet) {
+    box-shadow: inset 0 -1px 0 $govuk-border-colour;
+    display: flex;
+  }
+}
+
+.app-c-secondary-navigation__list {
+  list-style: none;
+  margin-bottom: govuk-spacing(4);
+  padding: 0;
+
+  @include govuk-media-query($from: tablet) {
+    box-shadow: none;
+    display: flex;
+    margin: 0;
+    white-space: nowrap;
+  }
+}
+
+.app-c-secondary-navigation__list-item {
+  box-shadow: inset 0 -1px 0 $govuk-border-colour;
+
+  @include govuk-media-query($from: tablet) {
+    display: flex;
+    margin-right: 20px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &:last-child {
+    box-shadow: none;
+  }
+
+  a,
```

Comment:
instead of `a` can we use a class name like `app-c-secondary-navigation__list-item-link`?

Reply:
Sure sounds good. I'l make the changes now.